Trials.com/react-frontend/src/components/reset-password.js

```diff
@@ -51,7 +51,7 @@ export default function ResetPassword() {
 		  </div>
 		</Card.Body>
 		<Card.Footer className="w-100 text-center align-bottom">
-			Need an account? <Link className="btn-link" to="/signup">Sign Up</Link>
+			Need an account? <Link className="btn-link" to="/signin">Sign Up</Link>
```

Comment:
ill probably make this another issue, right now the signin page is always /signin, maybe we can change it so when the login slides to the "signup" side, it can push the url to be signup instead, that way redirecting to signup actually makes more sense

Reply:
It does make sense to add this redirect because as of right now, the "need an account? sign up" link links back to the /signin path
![image](https://user-images.githubusercontent.com/49658348/102708829-f89df480-426a-11eb-8a7f-b73565aa954c.png)

Reply:
yeah it still is fine like this which is why i put it to /signin, i dont think it'll be used often if at all really so its not a big deal which is why i put TRIALS-29 as low priority, just a thought though. and if its easy enough itd be a good starting issue for someone too.